src/ui/actions/timeline.ts

```diff
@@ -526,7 +526,7 @@ export function playbackPoints(
       if (currentTime >= nextGraphicsTime) {
         try {
           let maybeNextGraphics = await Promise.race([nextGraphicsPromise, waitForTime(500)]);
-          if (!maybeNextGraphics) {
+          if (typeof maybeNextGraphics === "number") {
```

Comment:
I don't know how this worked in `main`. It looks broken to me. 🤷🏼

Reply:
It worked because `waitForTime()` used to return `Promise<void>` (whereas now it returns `Promise<number>`).

Reply:
Ah, right. That makes sense.